cockroachcloud/client-certs-dedicated.md

```diff
@@ -0,0 +1,403 @@
+
+Refer to [Transport Layer Security (TLS) and Public Key Infrastructure (PKI): The CockroachDB certificate Trust Store](../{{site.versions["stable"]}}/security-reference/transport-layer-security.html#the-cockroachdb-certificate-trust-store)
+
+{{site.data.alerts.callout_success}}
+The [Cluster Administrator](authorization.html#cluster-administrator) or [Org Administrator (legacy)](authorization.html#org-administrator-legacy) Organization role is required to manage the CA certificate for a {{ site.data.products.dedicated }} cluster.
+{{site.data.alerts.end}}
+
+<div class="filters clearfix">
+  <button class="filter-button page-level" data-scope="api">Using the API</button>
+  <button class="filter-button page-level" data-scope="tf">Using Terraform</button>
+</div>
+<p></p>
+
+<section class="filter-content" markdown="1" data-scope="api">
+
+1. Submit the asynchronous request, supplying your cluster ID, API key, and the path to the certificate JSON with your CA certificate, as described in [Create the certificate authority (CA) certificate](#create-the-certificate-authority-ca-certificate).
+
+    A `200` successful response code indicates that the asynchronous request was successfully submitted, but does not guarantee that the operation (configuring the CA certificate) successfully completed. You must confirm success with a follow-up `GET` request, as described in the next step.
+
+    {% include_cached copy-clipboard.html %}
+    ~~~shell
+    curl --request POST \
+      --url ${COCKROACH_SERVER}/api/v1/clusters/${CLUSTER_ID}/client-ca-cert \
+      --header "Authorization: Bearer ${API_KEY}" \
+      --header 'content-type: application/json' \
+      --data "@cockroach_client_ca_cert.json"
+    ~~~
+
+    ~~~txt
+    200 OK
+    ~~~
+
+1. Confirm success of the operation with the following `GET` request.
+
+    {% include_cached copy-clipboard.html %}
+    ~~~shell
+    curl --request GET \
+      --url ${COCKROACH_SERVER}/api/v1/clusters/${CLUSTER_ID}/client-ca-cert \
+      --header "Authorization: Bearer ${API_KEY}"
+    ~~~
+
+    `PENDING` indicates that the operation is still in process.
+    
+    ~~~txt
+    {
+      "status": "PENDING",
+      "x509_pem_cert": ""
+    }
+    ~~~
+    
+    `IS_SET` indicates that the operation completed successfully, confirming the configured public CA cert.
+
+    ~~~txt
+    {
+      "status": "IS_SET",
+      "x509_pem_cert": "-----BEGIN CERTIFICATE-----\nMIIDfzCCAmagAwIBAgIBADANBgkqhkiG9w0BAQ0FADBZMQswCQYDVQQGEwJ1czEL\nMAkGA1UECAwCV0ExDTALBgNVBAoMBHRlc3QxDTALBgNVBAMMBHRlc3QxEDAOBgNV\nBAcMB1NlYXR0bGUxDTALBgNVBAsMBHRlc3QwHhcNMjMwMzE2MjMyNTMxWhcNMjQw\n
+    ...\n-----END CERTIFICATE-----",
+    }
+    ~~~
+
+</section>
+<section class="filter-content" markdown="1" data-scope="tf">
+
+
+Add the following block to your Terraform template and apply the change:
```

Comment:
Maybe also link to the resource documentation here - https://registry.terraform.io/providers/cockroachdb/cockroach/latest/docs/resources/client_ca_cert?

Reply:
ok